packages/roosterjs-editor-plugins/lib/plugins/Paste/Paste.ts

```diff
@@ -94,6 +95,8 @@ export default class Paste implements EditorPlugin {
                 }
             } else if (fragment.querySelector(GOOGLE_SHEET_NODE_NAME)) {
                 sanitizingOption.additionalTagReplacements[GOOGLE_SHEET_NODE_NAME] = '*';
+            } else if (clipboardData.html && clipboardData.image) {
```

Comment:
It is a very common case that both html and image exist. So this check causes Line 100 cannot execute in a lot of cases.

Reply:
And since this will change existing paste behavior, suggest adding a new experimental feature so that we can turn it off anytime.